app/java/src/main/java/com/google/cloudclientapi/ProcessingJob.java

```diff
@@ -14,26 +14,93 @@
  * limitations under the License.
  */
 
+package com.google.cloudclientapi;
 
-//
-// ProcessingJob.java - take data in raw_data bucket,
-//                      process, and store in processed bucket.
-//
+import com.google.gson.Gson;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Dictionary;
+import java.util.Enumeration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-package com.google.cloudclientapi;
+public class ProcessingJob {
```

Comment:
nit: any reason not to declare this class as `static`?

Reply:
It looks like only nested classes (a class within a class) can be a `static` class in Java.
See this [StackOverflow post](https://stackoverflow.com/a/7486050).